src/main/webapp/ts/profileBuilder.ts

```diff
@@ -9,10 +9,10 @@ function setForm(value: string) {
             mentorForm.setAttribute('style','display:none;');
             userTypeButton.setAttribute('value','Mentee');
         }
-        else {
+        else{
```

Comment:
nit: I think there should be a space after else

Reply:
Done